ci/obd-demo.yaml

```diff
@@ -317,10 +317,10 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   annotations:
-    kubernetes.io/ingress.class: nginx
```

Comment:
this annotation is deprecated, it's replaced with the spec's field.

This is a benefit of not managing the deployments directly with Kardinal, we can see the WARNING deprecation messages from `kubectl`